src/expressions.md

```diff
@@ -133,96 +136,88 @@ assert_eq!(
 );
 ```
 
-> **Note**: Since this is applied recursively, these expressions are also
-> evaluated from innermost to outermost, ignoring siblings until there are no
-> inner subexpressions.
+> **Note**: Since this is applied recursively, these expressions are also evaluated from innermost to outermost, ignoring siblings until there are no inner subexpressions.
 
-## Place Expressions and Value Expressions
+## Expression categories
```

Comment:
Is there any way to keep old links to this section working? I know I linked to this section several times; I forgot if this was in other parts of the documentation (e.g. doc comments) or is GH issues / forum posts.

Reply:
You can add `<a name="place-expressions-and-value-expressions"></a>`  I think.

Reply:
The `name` attribute is deprecated. The span below with an `id` keeps the links working. I just tested this locally, and it scrolls to right below the chapter name, so I need to move the span to before the chapter name, but it otherwise works. We have links throughout the reference as well, and I didn't change them in this PR, yet.